app/component/itinerary/NavigatorIntro/NavigatorIntro.js

```diff
@@ -0,0 +1,94 @@
+import Button from '@hsl-fi/button';
+import { connectToStores } from 'fluxible-addons-react';
+import PropTypes from 'prop-types';
+import React from 'react';
+import { FormattedMessage, intlShape } from 'react-intl';
+import { configShape } from '../../../util/shapes';
+import Icon from '../../Icon';
+import NavigatorIntroFeature from './NavigatorIntroFeature';
+
+const NavigatorIntro = (
+  { logo, onPrimaryClick, onClose, isLoggedIn },
+  context,
+) => {
+  const { config, intl } = context;
+
+  const primaryColor =
+    config.colors?.accessiblePrimary || config.colors?.primary || 'black';
+
+  return (
+    <div className="navigator-intro-modal-content">
+      <div className="body">
+        {logo && <img src={logo} alt="navigator logo" />}
+        <FormattedMessage tagName="h2" id="navigation-intro-header" />
+        <div className="navigation-intro-body">
+          <NavigatorIntroFeature
+            icon="icon-icon_future-route"
+            iconColor={primaryColor}
+            iconBackgroundColor={config.colors?.backgroundInfo || 'transparent'}
```

Comment:
Maybe `|| tranparent` is not needed. Base config sets the color.

Reply:
Will fix.